src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String STRING_SEPARATOR = ",";
+    private static final String SHOP_SUPPLIER = "supply";
+    private static final String SHOP_BUY = "buy";
+    private static final String RESULT_COUNT = "result";
+    private static final int ARGUMENT = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String storetable = readDataFromFile(fromFileName);
+        String report = filterData(storetable);
+        writeNewData(report, toFileName);
+    }
 
+    private String readDataFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                stringBuilder.append(line).append(System.lineSeparator());
+                line = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file" + fromFileName, e);
+        }
+        return stringBuilder.toString();
+    }
+
+    private String filterData(String storetable) {
```

Comment:
filterData means the method filters sth, but your method calculates data. Rename this method